automation/JJB/python_analysis.yaml

```diff
@@ -1,5 +1,5 @@
 - job:
-    name: sonarqube_pulp_python_full_analysis
+    name: sonarqube_python_full_analysis
```

Comment:
Slight nit: I recommend a consistant naming scheme, so this would be "sonarqube_python_analysis".